src/tjenesteomrader/index.tsx

```diff
@@ -0,0 +1,66 @@
+import Layout from 'src/layout';
+import style from './index.module.css';
+import digitalTjenesteBlob from './images/indexBlobs/digitalTjeneste.svg';
+import serviceBlob from './images/indexBlobs/strategi.svg';
+import datadrevet from './images/indexBlobs/datadrevet.svg';
+import kultur from './images/indexBlobs/kultur.svg';
+import Entrance from './components/entrance/entrance';
+
+const Tjenesteomrader = () => {
+  return (
+    <Layout>
+      <div className={style.tjenesteomrader}>
```

Comment:
Jeg ville lagd hele dette som en komponent ettersom den også skal brukes på fremsiden for å erstatte dette:
<img width="1024" alt="Screenshot 2022-11-24 at 19 58 25" src="https://user-images.githubusercontent.com/606374/203850949-e4ab292b-1d57-4fd0-ae77-e63618e8cb28.png">